core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -329,7 +331,7 @@ static uint32_t desc_to_mattr(unsigned level, uint64_t desc)
 	if (desc & UPPER_ATTRS(PXN))
 		a &= ~TEE_MATTR_PX;
 
-	COMPILE_TIME_ASSERT(ATTR_DEVICE_INDEX == TEE_MATTR_MEM_TYPE_DEV);
+	COMPILE_TIME_ASSERT(ATTR_DEVICE_nGnRE_INDEX == TEE_MATTR_MEM_TYPE_DEV);
```

Comment:
Please add a compile time assert for `TEE_MATTR_MEM_TYPE_STRONGLY_O` too.

Reply:
Ack